examples/ticks/ticks_too_many.py

```diff
@@ -0,0 +1,68 @@
+"""
+=====================
+Fixing too many ticks
+=====================
+
+One common cause for unexpected tick behavior is passing a list of strings
+instead of numbers or datetime objects. This can easily happen without notice
+when reading in a comma-delimited text file. Matplotlib treats lists of strings
+as "categorical" variables
+(:doc:`/gallery/lines_bars_and_markers/categorical_variables`), and by default
+puts one tick per "category", and plots them in the order in which they are
+supplied.  If this is not desired, the solution is to convert the strings to
+a numeric type as in the following examples.
+
+"""
+
```

Comment:
Using sections for the examples structures the content and announces what you'll see in that code blocki

```suggestion

############################################################################
# Example 1: Strings can lead to an unexpected order of number ticks
# ------------------------------------------------------------------

```